### ISSUE TITLE
Decreased CSF/serum albumin ratio

### ISSUE THREAD

**Issue by NuriaQueralt** [obophenotype/human-phenotype-ontology#2287]
>This is a request for a new term

>1. Preferred term label (e.g., Asplenia)_
Decreased CSF/serum albumin ratio

>2. Synonyms (e.g., Absent spleen)
Low CSF/serum albumin ratio

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID to refer to any relevant article that provides additional information about the suggested term).
CSF/serum albumin ratio is below the lower limit of normal

>4. Reference (PMID)
This reference relates this requested phenotype with the NGLY1-deficiency rare disease
https://www.ncbi.nlm.nih.gov/pubmed/27388694

>5. Suggested parent term 
http://compbio.charite.de/hpoweb/showterm?id=HP:0002921

>6. If you know of a disease that is characterized by this feature, please indicate it (e.g., by its Orphanet or OMIM id), and we will add the annotation to our database.
DO: DOID:0060728
ORPHANET: ORPHA:404454
OMIM:615273

>7. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as HPO:probinson (organization, first initial-last name)
http://orcid.org/0000-0003-0169-8159

**Comment by pnrobinson** [comment]
Question --  An increased CSF/serum albumin ratio is a sign of blood brain barrier leakiness. As far as I know a decrease in this ratio is harder to interpret. Are you sure about "decreased"?  I do not now have access to this paper, could you send it to me via email peter.robinson@jax.org

**Comment by pnrobinson** [comment]
In fact, https://www.ncbi.nlm.nih.gov/pubmed/27388694 does state that the CSF/serum albumin ratio was low in patients with CONGENITAL DISORDER OF DEGLYCOSYLATION owing to NGLY1 variants, so we do need both terms.

**Comment by NuriaQueralt** [comment]
Question -- i can not see the HP_ID created for this concept, could you help me?  i only see the new IRI created based on the string name: "http://purl.obolibrary.org/obo/hp.owl#Decreased_CSF/serum_albumin_ratio"

**Comment by nicolevasilevsky** [comment]
@NuriaQueralt 
here you go: http://purl.obolibrary.org/obo/HP_0030978

**Comment by NuriaQueralt** [comment]
@nicolevasilevsky Thanks!
